utils/httpUtils.ts

```diff
@@ -72,3 +71,111 @@ export const notFound = (message: string) => {
     message: message,
   });
 };
+
+export const getFeatureFlagsFromRequest = context => {
+  return context.request.headers.get("x-should-read-from-nextgen");
+};
+
+export const shouldReadFromNextGen = async context => {
+  let shouldReadFromNextGen = getFeatureFlagsFromRequest(context);
+  if (shouldReadFromNextGen === true || shouldReadFromNextGen === "true" || shouldReadFromNextGen === "True") {
+    // if the header is set, return the value
+    return true;
+  }
+  return false;
+};
+
+export const formatFedQueryForNextGen = (query: string) => {
+  let cleanQuery = query;
+  // remove fed Prefix
+  if (query.indexOf("fed") !== -1) {
+    const splitQueryOnFed = query.split("fed");
+    const firstLetterLowerCase = splitQueryOnFed[1][0].toLowerCase();
+    splitQueryOnFed[1] = splitQueryOnFed[1].slice(1);
+    splitQueryOnFed.splice(1, 0, firstLetterLowerCase);
+    cleanQuery = splitQueryOnFed.join("");
+  }
+
+  // remove input object from variables
+  const finishedQuery = cleanQuery
+    .replace("input: {", "")
+    .replace("}}", "}")
+    // TODO: (suzette 02/27/24) FIX THESE HACKS TO MAKE THIS FUNCTION MORE UNIVERSAL
```

Comment:
This function is a little lame at the moment. I am hoping it can mature as we add more connections.